### DISCUSSION TITLE
How to use negative value?

### DISCUSSION THREAD

**Discussion by gilvex** [Mokshit06/typewind#36] [Q&A] [answered]
Hey! How to use negative value for example ` -mt-1 ` ?

**Comment by Mokshit06** [comment, marked as answer]
Hey! I didn't know tailwind supported negative values. Will add it. The syntax would be `tw._mt_1`

**Reply by gilvex** [reply]
Cool! Ty!

**Reply by Mokshit06** [reply]
This should work in v0.1.1!
